sys-kernel/linux-headers/linux-headers-3.18.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="4"
```

Comment:
Please don't change EAPI of existing stable ebuilds. This requires revbump.

Reply:
```
$ equery meta sys-kernel/linux-headers
Maintainer:  toolchain@gentoo.org
Upstream:    Remote-ID:   cpe:/o:linux:linux_kernel ID: cpe
Homepage:    https://www.kernel.org/
Homepage:    https://www.gentoo.org/
Location:    /var/db/repos/gentoo/sys-kernel/linux-headers
Keywords:    3.18:0: 
Keywords:    4.4:0: 
Keywords:    4.9:0: 
Keywords:    4.13:0: alpha amd64 arm arm64 hppa ia64 m68k ppc ppc64 s390 sh sparc x86
Keywords:    4.14:0: 
Keywords:    4.15:0: 
Keywords:    4.16:0: 
Keywords:    4.16-r1:0: ~alpha ~amd64 ~amd64-linux ~arm ~arm-linux ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-linux
License:     GPL-2
```
Unless I'm mistaken, this indicates that the 3.18 version is not 'stable' as far as keywording
goes.

Reply:
No, it is. The lack of keywords on those lines indicates that they match the keywords farther down. Kind of like a switch/case fallthrough.

See the output of eshowkw linux-headers for confirmation.

Reply:
So, everything 4.13:0 and above all have the same keywords as 4.13, and
everything 4.16-r1 - 4.14 has the same keywords? That is a bit funky imho.

Reply:
I don't know if by "above" you mean "versions greater than" or "appearing above in this output"

Regardless, you can answer these questions by looking at it locally. Consecutive versions having the same keywords doesn't seem strange at all.